internal/security/publickey.go

```diff
@@ -0,0 +1,67 @@
+package security
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"fmt"
+	"io"
+)
+
+// PublicKey публичный RSA ключ. используется для штфрования данных.
+type PublicKey *rsa.PublicKey
```

Comment:
Please avoid global variables.

Reply:
This is not a global variable, it is a type declaration